lms/templates/fields/field_dropdown.underscore

```diff
@@ -1,13 +1,11 @@
-<% if (editable !== 'never') { %>
-    <% if (title && titleVisible) { %>
-        <label class="u-field-title" for="u-field-select-<%- id %>">
-            <%- title %>
-        </label>
-    <% } else { %>
-        <label class="sr" for="u-field-select-<%- id %>">
-            <%- screenReaderTitle %>
-        </label>
-    <% } %>
+<% if (title && titleVisible) { %>
```

Comment:
Update:

Checked all the instances of these dropdowns and it appears as if the 'titleVisible' was the original override to show or hide the title, added on 5/4/15 and the editable=never was added in september of that year, seemingly without noticing the purpose of the titleVisible field. 

Pretty much, two people went with two philosophies on when to display the title. The fix seems to be to check if the editable !== never or the title is visible, adhering to both. Will change now.